samples/zigbee/template/prj_nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
I just wonder, this is pretty similar to the default `prj.conf` - did you consider creating `boards/nrf5340dk_nrf5340_cpuapp.conf` and include only differences in there? Such a config file should be applied automatically for that specifc board.

Reply:
I'll try that. Thanks for the suggestion!